clients/java/pom.xml

```diff
@@ -278,7 +278,7 @@
               <!-- https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/java.md -->
               <generatorName>java</generatorName>
               <inputSpec>${openapi.dir}/rest-api.yaml</inputSpec>
-              <modelPackage>io.camunda.zeebe.client.protocol.rest</modelPackage>
+              <modelPackage>io.camunda.client.protocol.rest</modelPackage>
```

Comment:
❓ I changed also the package of those classes but I saw that are used also from other product (eg Tasklist), I think we can't move those generated classes for now

Reply:
At least for the client-generated REST data model, this should not be used outside the Java client. There are gateway counterparts, too. Maybe you saw those being used?

Reply:
Like here: https://github.com/camunda/camunda/blob/73653beb549ae7bb0cae5277c8354005d486e3a6/tasklist/webapp/src/test/java/io/camunda/tasklist/webapp/service/TaskServiceTest.java#L52

Reply:
Interesting. Well, they actually don't use that import much. They create an instance but don't use it 😅

Reply:
Can I simply remove it, what do you think ?

Reply:
It looks like you can, yes. The test should pass.